packages/react-charting/src/components/HorizontalBarChartWithAxis/HorizontalBarChartWithAxisRTL.test.tsx

```diff
@@ -315,7 +315,9 @@ describe('Horizontal bar chart with axis - Subcomponent Labels', () => {
       expect(getById(container, /showDots/i)[0]!.textContent!).toEqual('Stri...');
```

Comment:
## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=30218&commitId=acdb183f4c8073ba0a0565c1014d6b6f6fa583b9&env=prod&clientType=FLUENTUI)


  <details><summary><b>PeoplePicker</b> 7 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|PeoplePicker.Normal.suggestions.chromium.png|12| Changed|
|PeoplePicker.Normal Overflow selected.default.chromium.png|0| Removed|
|PeoplePicker.Normal Overflow selected.suggestions.chromium.png|0| Removed|
|PeoplePicker.Normal selected - RTL.default.chromium.png|0| Removed|
|PeoplePicker.Normal selected - RTL.suggestions.chromium.png|0| Removed|
|PeoplePicker.Normal selected.default.chromium.png|0| Removed|
|PeoplePicker.Normal selected.suggestions.chromium.png|0| Removed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>

Reply:
Unrelated change.